circuits/cpp/barretenberg/cpp/src/barretenberg/proof_system/flavor/flavor.hpp

```diff
@@ -250,25 +250,6 @@ template <class FF, typename Tuple, std::size_t Index = 0> static constexpr auto
     }
 }
 
-/**
```

Comment:
This is unrelated to this work but was unused and should have been removed previously.